Alarm system no more working after MySQL drivers updated to ver. 8.

After the MySQL drivers update the following exception happened when opening the Alarm perspective:

```
Loading class `com.mysql.jdbc.Driver'. This is deprecated. The new driver class is `com.mysql.cj.jdbc.Driver'. The driver is automatically registered via the SPI and manual loading of the driver class is generally unnecessary.
2018-08-29 11:53:57.381 SEVERE [Thread 20] org.csstudio.alarm.beast.ui.clientmodel.AlarmClientModel (readConfiguration) - Cannot connect to RDB
java.lang.Exception: Cannot connect to database, No suitable driver found for jdbc:mysql://172.16.107.12/alarm
        at org.csstudio.alarm.beast.client.AlarmConfiguration.<init>(AlarmConfiguration.java:98)
        at org.csstudio.alarm.beast.client.AlarmConfiguration.<init>(AlarmConfiguration.java:125)
        at org.csstudio.alarm.beast.ui.clientmodel.AlarmClientModel$1.<init>(AlarmClientModel.java:398)
        at org.csstudio.alarm.beast.ui.clientmodel.AlarmClientModel.readConfiguration(AlarmClientModel.java:398)
        at org.csstudio.alarm.beast.ui.clientmodel.ReadConfigJob.run(ReadConfigJob.java:49)
        at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
Caused by: java.sql.SQLException: No suitable driver found for jdbc:mysql://172.16.107.12/alarm
        at java.sql.DriverManager.getConnection(DriverManager.java:689)
        at java.sql.DriverManager.getConnection(DriverManager.java:208)
        at org.csstudio.platform.utility.rdb.internal.MySQL_RDB.connect(MySQL_RDB.java:63)
        at org.csstudio.platform.utility.rdb.RDBUtil.<init>(RDBUtil.java:179)
        at org.csstudio.platform.utility.rdb.RDBUtil.connect(RDBUtil.java:143)
        at org.csstudio.alarm.beast.client.AlarmConfiguration.<init>(AlarmConfiguration.java:94)
        ... 5 more
```

--- a/applications/alarm/alarm-plugins/org.csstudio.alarm.beast/preferences.ini
+++ b/applications/alarm/alarm-plugins/org.csstudio.alarm.beast/preferences.ini
@@ -9,6 +9,9 @@ readonly=false
 allow_config_selection=true
 
 # RDB that holds configuration and snapshot of state
+# Depending on MySQL Server version, might need additional parameters
+# like ?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC
+# rdb_url=jdbc:mysql://localhost/ALARM?serverTimezone=UTC
 rdb_url=jdbc:mysql://localhost/ALARM
 rdb_user=alarm
 rdb_password=$alarm

--- a/core/platform/platform-plugins/org.csstudio.platform.utility.rdb/META-INF/MANIFEST.MF
+++ b/core/platform/platform-plugins/org.csstudio.platform.utility.rdb/META-INF/MANIFEST.MF
@@ -8,7 +8,7 @@ Export-Package: org.csstudio.platform.utility.rdb
 Bundle-ClassPath: .
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-Description: Helper for connecting to various RDB dialects
-Import-Package: com.mysql.jdbc,
+Import-Package: com.mysql.cj.jdbc,
  org.postgresql,
  oracle.jdbc.driver,
  oracle.jdbc

--- a/core/platform/platform-plugins/org.csstudio.platform.utility.rdb/src/org/csstudio/platform/utility/rdb/internal/MySQL_RDB.java
+++ b/core/platform/platform-plugins/org.csstudio.platform.utility.rdb/src/org/csstudio/platform/utility/rdb/internal/MySQL_RDB.java
@@ -59,7 +59,7 @@ public class MySQL_RDB implements RDBImpl
             // but now normal driver can handle replication
         }
         // Class loader locates the plain MySQL driver
-        Class.forName("com.mysql.jdbc.Driver").newInstance();
+        Class.forName("com.mysql.cj.jdbc.Driver").newInstance();
         connection = DriverManager.getConnection(url, props);
 
         // Basic database info

